content/tutorials/reactor-event-ifttt.textile

```diff
@@ -0,0 +1,196 @@
+In this specific example we'll be using IFTTT to trigger a post to a "Slack channel":https://ifttt.com/slack every time a message is published on your Ably channel.
+
+h2. What we'll be building
+
+We'll be building a bridge simulator game using HTML, CSS, JavaScript and "Reactor Functions":https://www.ably.io/documentation/general/functions. Players will receive commands from their "captain" via Slack. These commands will let them know what to press in the game to move to the next command. Clicking the controls that match the commands will win them the game, mistakes will lose and crash the spaceship.
+
+
+<a href="/images/tutorials/reactor/reactor-event-ifttt-game.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-ifttt-game.png" style="width: 100%" alt="screenshot of the game we will be making">
+</a>
+
+h2. Prerequisite Accounts
+
+You'll need an Ably account, login or sign up for free at "https://www.ably.io/":https://www.ably.io/.
+You'll also need an IFTTT account, login or sign up at "https://ifttt.com/":https://ifttt.com/.
+Finally you'll need an account with Slack and a workspace to send messages to, sign in or create a workspace at "https://slack.com/signin":https://slack.com/signin
+
+h2. Setup
+
+We're going to need an Ably API key for use by your browser application. For our purposes, only one API key is required. If you extend the tutorial with your own functionality (you may wish to create a separate API key for the IFTTT if you need it to have a different set of permissions than those assigned for use by the browser app).
+
+First, log into your Ably dashboard, find the app you're going to be using for this tutorial (the "Sandbox" app is probably fine), then click "Manage App".
+
+<a href="/images/tutorials/reactor/reactor-event-ably-apikey-1.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-ably-apikey-1.png" style="width: 100%" alt="Manage your Ably app">
+</a>
+
+Select the "*API Keys*" tab, and click "*Create a new API key*":
+
+<a href="/images/tutorials/reactor/reactor-event-ably-apikey-2.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-ably-apikey-2.png" style="width: 100%" alt="Create a new API key">
+</a>
+
+# Enter a name for your API key
+# Select only the "publish" and "subscribe" options
+# Click the "Create key" button
+
+Scroll down if necessary, and copy and paste your new API key to a text file for later use in the application.
+
+h2. Creating an Ably Reactor Rule
+
+We'll need to create a Reactor Event Rule in your Ably dashboard. This is what will trigger the IFTTT event.
+
+# Go to your Ably Dashboard
```

Comment:
Ably Dashboard could be a link to the generic account dashboard.  We have support for `any` in the URL, which @tomczoink can tell you about. @tomczoink I think we should add a mention of how any works in https://docs.ably.io/client-lib-development-guide/documentation-formatting-guide/ btw. for future people so that this knowledge is not kept in our heads. WDYT?